subsys/bluetooth/controller/hci/hci.c

```diff
@@ -6529,6 +6529,7 @@ static void le_per_adv_sync_report(struct pdu_data *pdu_data,
 	uint8_t cte_type = BT_HCI_LE_NO_CTE;
 	struct pdu_adv_com_ext_adv *p;
 	struct pdu_adv_ext_hdr *h;
+	struct ll_sync_set *sync;
```

Comment:
Given that `sync` is only used once to check a value, consider making this `const`

```suggestion
	const struct ll_sync_set *sync;
```

Reply:
done